Bottom: b678a9ed9dc3227fccde704307ecb808cc1b44b4
Top:    04edbd9d3f63cc91346e1ef85f0043e728dd71c4
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2016-02-18 20:35:48 +0000

Add warning about bogus code in strflags.c


---

diff --git a/src/strflags.c b/src/strflags.c
index 3e57696..d6d6273 100644
--- a/src/strflags.c
+++ b/src/strflags.c
@@ -576,6 +576,7 @@ dump_flag (FlagType * f)
 {
   int l;
   printf ("F:%08x T:[", f->f);
+#warning THIS NEXT LINE IS BOGUS, THE ARRAY IS HALF THIS LENGTH
   for (l = 0; l < (MAX_LAYER + 7) / 8; l++)
     printf (" %02x", f->t[l]);
   printf ("]");
